src/controllers/scoresController.ts

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import { util } from '../config/gameInfoUtil';
+import pg from 'pg';
+import { parse } from 'pg-connection-string';
+require('dotenv').config();
+
+const dbUrl: string = process.env.DB_URL || '';
+const config: object = parse(dbUrl);
+const pool = new pg.Pool(config);
+pool.connect();
+
+export const scoresController = {
+  getScores(req: express.Request, res: express.Response) {
+    const gameName: string = req.params['game_name'];
+    const gameIndex: number = util.gameToIndex(gameName);
+    const scores = getScoresFromDB(gameIndex);
+    res.render('scores.pug', { scores, gameName });
```

Comment:
Do `scores` and `gameName` need to be dict keys here?

Reply:
No. In JavaScript ES6, single value in a dictionary is shorthand for a key and a value that are the same. If you run `tslint` it should warn you about `{ scores: scores, gameName: gameName }` being redundant, and `npm run fix` should automatically fix it.